packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -65,6 +65,43 @@ import 'theme.dart';
 /// ```
 ///
 ///{@end-tool}
+///
+/// {@tool snippet}
```

Comment:
The dartdoc analyzer is reporting an error (see the Checks section for more info, but here is a link to the error: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8851609791888740576/+/steps/prepare_environment/0/steps/Analyze/0/steps/analyze/0/stdout):

```
▌11:31:40▐ RUNNING: cd .; bin/cache/dart-sdk/bin/dart /b/s/w/ir/k/flutter sdk/dev/bots/analyze_sample_code.dart
Found 350 snippet code blocks, 16 sample code sections, and 202 dartpad sections.
Starting analysis of code samples.
src/material/reorderable_list.dart:93:13:     setState(() {
>>> error: The function 'setState' isn't defined (undefined_function)
```

Which is due to the fact that this code assumes it is part of a stateful widget (it is calling `setState`). You can fix this by changing the `@tool` command to something like:

```
/// {@tool dartpad --template=stateful_widget_material}
```

Which provides a template that includes a stateful widget.

Reply:
added the stateful widget annotation
```
/// {@tool dartpad --template=stateful_widget_material}
```